docs/book/src/reference/platform.md

```diff
@@ -0,0 +1,240 @@
+# Platform Supported
+
+Kubebuilder produces solutions that by default can work on multiple platforms or specific ones, depending on how you 
+build and configure your workloads. This guide aims to help you properly configure your projects according to your needs.
+
+## Overview
+
+To provide support on specific or multiple platforms, you must ensure that all images used in workloads are built to 
+support the desired platforms. Note that may not be the same as the platform where you develop your solutions 
+and use KubeBuilder, but instead the platform(s) where your solution should run and be distributed. 
+A good recommendation would be to build solutions that work on multiple platforms so that your project could still 
+work no matter the operating system and architecture of the Kubernetes cluster where it is installed .
```

Comment:
Nit:
```suggestion
It is recommended to build solutions that work on multiple platforms so that your project works
on any Kubernetes cluster regardless of the underlying operating system and architecture.
```